tests/testthat/test_featsel_filters.R

```diff
@@ -72,3 +72,16 @@ test_that("randomForestSRC_var.select filter handles user choices correctly", {
     more.args = list("randomForestSRC_var.select" = c(method = "vh.imp"))))
   )
 })
+
+
+test_that("Ensemble filters can accept duplicate base methods", {
+  lda = makeLearner(cl = "classif.lda", id = "lda_class", predict.type = "response")
+  ff = filterFeatures(multiclass.task, 
+                      method = "E-mean", 
+                      base.methods = list("filter1" = "univariate.model.score", "filter2" = "univariate.model.score"),
+                      abs = 2,
+                      more.args = list("filter1" = list(perf.learner = lda), "filter2" = list(perf.learner = lda))
```

Comment:
Should we also assert that `more.args` is a named list?

Reply:
There is already such an assertion in generateFilterValues.R, line 94:
`  assertList(more.args, names = "unique", max.len = length(method))
`

Reply:
Just FYI: You can cross-link code by clicking on the line number of a file on Github and then select "copy permalink". This way the code line are embedded and you do not need to c/p :)

Reply:
Thanks for the tip.